services/submission.js

```diff
@@ -1654,7 +1654,7 @@ const sendEmails = {
                     submissionName: `${aSubmission?.name},`,
                     dataCommonsName: aSubmission?.dataCommons,
                     contactName: `${aSubmission?.conciergeName || 'NA'}`,
-                    contactEmail: `${aSubmission?.contactEmail || 'NA'}.`
+                    contactEmail: `${aSubmission?.conciergeName || 'NA'}.`
```

Comment:
Why is conciergeName used for both of these fields here? Is this correct?

Reply:
My apology. This should be conciergeEmail. I have pushed the change.